jbpm-installer/src/test/java/org/jbpm/persistence/scripts/TestPersistenceContext.java

```diff
@@ -119,31 +129,27 @@ public void clean() {
      * @param scriptsRootFolder Root folder containing folders with SQL scripts for all supported database systems.
      * @throws IOException
      */
-    public void executeScripts(final File scriptsRootFolder) throws IOException, SQLException {
-        executeScripts(scriptsRootFolder, null);
+    public void executeScripts(final File scriptsRootFolder, boolean dropFilesExcluded) throws IOException, SQLException {
+        executeScripts(scriptsRootFolder, dropFilesExcluded, null);
     }
     
-    public void executeScripts(final File scriptsRootFolder, String type) throws IOException, SQLException {
+    public void executeScripts(final File scriptsRootFolder, boolean dropFilesExcluded, String type) throws IOException, SQLException {
         testIsInitialized();
-        final File[] sqlScripts = TestsUtil.getDDLScriptFilesByDatabaseType(scriptsRootFolder, databaseType, true);
+        final File[] sqlScripts = TestsUtil.getDDLScriptFilesByDatabaseType(scriptsRootFolder, databaseType, true, dropFilesExcluded);
+        if (sqlScripts.length == 0 && databaseType!=DERBY && dropFilesExcluded) {
```

Comment:
no, it's only for create scripts (&& dropFiles condition in the clause); in case there're no sql scripts, throw a meaningful exception (this is valid for the case of upgrading scripts for sybase, that doesn't contain ddl60 scripts).

Reply:
So what if there are no drop scripts, are we OK with that?